packages/components/src/TemporaryAddressChange/Component.purs

```diff
@@ -96,7 +96,7 @@ component = React.createComponent "TemporaryAddressChange"
 calcMinEndDate :: Maybe DateTime -> Maybe DateTime
 calcMinEndDate Nothing = Nothing
 calcMinEndDate (Just startDate) = do
-  let week = Time.Duration.Days 7.0
+  let week = Time.Duration.Days 6.0
```

Comment:
Technically this is not a week anymore 😆 maybe you can call it something like `interval` or `days` (I don't  really know) and add a comment on why it's `6.0`.

Reply:
I think `week` is a good name here as that is the result of the action we are doing here and conveys the intent better than just `interval` or `days`

Reply:
I'd rather add a comment to explain that we want the minimum duration to be one week

Reply:
So, could we find a solution here? Customer service is anxious.

Reply:
Feel free to merge at any time